material-card/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@ohos/material-card",
+  "version": "1.0.0",
+  "description": "",
+  "main": "src/materialCard.ets",
+  "scripts": {
```

Comment:
convert material-card to a module, if hyphen is not support, then change name to materialcard